command-line-local-shortener/main.go

```diff
@@ -107,10 +107,12 @@ func main() {
 
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
 	router.GET("/:id", getReq)
-
-	router.GET("/home", func(c *gin.Context) {
-		c.JSON(200, gin.H{"hello": "world"})
+	router.GET("/", func(c *gin.Context) {
+		c.HTML(http.StatusOK, "index.tmpl", gin.H{
+			"title": "Main website",
```

Comment:
Shouldn't need a title here.... this is what the `<title>....</title>` tag is for in the `html` file

Reply:
can i just leave this blank then?

Reply:
Yes, that should be fine